rollup.config.mjs

```diff
@@ -15,7 +15,6 @@ export default {
     license({
       banner: {
         content: {
-          // eslint-disable-next-line no-undef
           file:     path.join(__dirname, "LICENSE"),
```

Comment:
Lets use
```suggestion
          file:     path.join(import.meta.dirname, "LICENSE"),
```
like in the extension project.

Reply:
Thanks for the reminder